extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -249,4 +266,35 @@ export class ConsentInstance {
       return CONSENT_ITEM_STATE.UNKNOWN;
     });
   }
+
+  /**
+   * send a POST request to the updateHref with userId with fixed scope
+   * and consentInstanceId
+   * @param {boolean} state
+   */
+  sendUpdateHrefRequest_(state) {
+    if (!this.onUpdateHref_) {
+      return;
+    }
+    const cidPromise = Services.cidForDoc(this.ampdoc_).then(cid => {
+      return cid.get({scope: CID_SCOPE, createCookieIfNotPresent: true},
+          Promise.resolve());
+    });
+    cidPromise.then(userId => {
+      const request = /** @type {!JsonObject} */ ({
+        'consentInstanceId': this.id_,
+        'ampUserId': userId,
+        'consentState': state,
+      });
+      const init = {
+        credentials: 'include',
+        method: 'POST',
+        body: request,
+        requireAmpResponseSourceOrigin: false,
```

Comment:
this is deprecated. use `ampCors: false`

Reply:
done